modules/guides/pages/bloblang/methods.adoc

```diff
@@ -1867,6 +1866,27 @@ root.id = this.id.string()
 # Out: {"id":"228930314431312345"}
 ```
 
+=== `timestamp`
+
+Attempts to parse a value into a `timestamp`. The input value must be one of the following:
+
+- An integer that represents a Unix timestamp (in seconds)
+- A floating-point number that represents a Unix timestamp (in seconds)
+- A string in the RFC 3339 format
+
+You can also provide an optional argument, which is returned when the value cannot be parsed into a `timestamp`. The output of this method can be further transformed using <<timestamp-manipulation,timestamp manipulation methods>>.
+
+==== Parameters
+
+*`default`* <(optional) timestamp> An optional value to return if the target value cannot be parsed as a `timestamp`.
+
+==== Examples
+
+```coffeescript
+root.foo = this.ts.timestamp()
+root.bar = this.none.timestamp(1234567890.timestamp())
```

Comment:
It may be worth moving away from `foo` and `bar` as examples. They aren't descriptive and provide no value.

```suggestion
root.creation_time = this.ts.timestamp()
root.default_time = this.none.timestamp(1734443120.timestamp())
```

Reply:
Updated.